attributes/default.rb

```diff
@@ -18,23 +18,104 @@
 #
 
 case node['platform_family']
-when 'smartos', 'rhel', 'debian', 'fedora', 'mac_os_x', 'suse', 'amazon'
+when 'smartos', 'rhel', 'debian', 'fedora', 'mac_os_x', 'suse', 'amazon', 'windows'
   default['nodejs']['install_method'] = 'package'
 else
   default['nodejs']['install_method'] = 'source'
 end
 
-default['nodejs']['version'] = '6.10.2'
+default['nodejs']['engine'] = 'node'
+default['nodejs']['version'] = '6.11.2'
+default['nodejs']['checksum']['6.11.2']['tar.gz'] = '20146ed51b638404665737ed8a25cc06e96d7d7259eb90a4bdec4730a78002a6'
```

Comment:
I'm concerned that this attribute structure is something we're never going to be able to maintain. It might make more sense to either skip the checksum and rely on the checksum provided by the http server or parse out https://nodejs.org/dist/v6.11.2/SHASUMS256.txt.asc for the appropriate checksum.

Reply:
I totally agree with you. I was hoping to this poor mans solution in and then improve upon it as they are loads of people that need Windows now. 

Any idea how to implement this? Is there a cookbook that does something like this already?